docs/content/docs/advanced/summarizer.md

```diff
@@ -19,6 +19,25 @@ graph LR
   A --> C1([DataStore_1])
   A --> C2([DataStore_2])
   A --> CH([.chunks])
+  classDef tree fill:#faa,stroke:#000
+  class A,B,S,C1,C2,D1,D2,ST tree
+{{< /mermaid >}}
+
+<span class="screen-reader-text">
+<ul aria-label="Describes the hierarchical structure of the summary tree from the root to the first layer of leaf nodes.">
```

Comment:
These aren't being read off by narrator. It goes from the above paragraph into the list itself

> The root node corresponds to the Container, and it contains the protocol information as well as any partially processed chunk op data.<mark name="ContinuousReadingEnd Id:3"/>
Enter List, <say-as interpret-as="spell">1</say-as> of <say-as interpret-as="spell">1</say-as> level <say-as interpret-as="spell">1</say-as>, <voice xml:lang="en-US">bullet</voice>, root<mark name="ContinuousReadingEnd Id:4"/>
<say-as interpret-as="spell">1</say-as> of <say-as interpret-as="spell">5</say-as> level <say-as interpret-as="spell">2</say-as>, <voice xml:lang="en-US">hollow bullet</voice>, .protocol<mark name="ContinuousReadingEnd Id:5"/>
<say-as interpret-as="spell">2</say-as> of <say-as interpret-as="spell">5</say-as> level <say-as interpret-as="spell">2</say-as>, <voice xml:lang="en-US">hollow bullet</voice>, _scheduler<mark name="ContinuousReadingEnd Id:6"/>

Reply:
Since we're already hiding this section, this aria-label would probably do better as a paragraph tag on it's own.

Reply:
a label and labeledby works really well

![image](https://user-images.githubusercontent.com/1434956/95260132-149e0800-07dd-11eb-9da4-173759396b9c.png)

Reply:
Nice! Thanks for the suggestion. Will address in the next iteration.